solutions/devsprint-michael-douglas-2/DeliveryApp/Screens/Components/AddressView.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  AddressView.swift
+//  DeliveryApp
+//
+//  Created by Alexandre Cardoso on 21/02/22.
+//
+
+import UIKit
+
+protocol AddressViewDelegate: AnyObject {
+    func didEditAddress()
```

Comment:
Considerando que esse método do Delegate é um método de uma ação de interação, podemos utilizar esse nome:
```suggestion
    func didTapEditAddress()
```